### ISSUE TITLE
Is there a way to replace the camera by our own implementation?

### ISSUE THREAD

**Issue by arqlz** [ParkSangGwon/TedImagePicker#94]
The reason im looking out for this is because in some Samsung devices there is a delay between the time the image is taken and the then time its presented on the gallery, im looking for a custom implementation with cameraX for a lower resolution (1280) for those devices.

**Comment by abkoradiya** [comment]
Yes need this feature. Becuase In my case my app will restart when open camera due to camera requrie high ram and system decide to kill my app.
